src/encoded/audit/analysis.py

```diff
@@ -146,6 +417,180 @@ def audit_experiment_standards_dispatcher(value, system, files_structure):
+        yield from check_analysis_long_read_rna_standards(
+            value,
+            files_structure,
+            ['Long read RNA-seq pipeline'],
+            '/rna-seq/long-read-rna-seq/'
+        )
+
+    '''
+    ChIP-seq analyses
+    '''
+
+    if any(pipeline['title'] in [
+            'ChIP-seq read mapping',
+            'Histone ChIP-seq 2',
+            'Histone ChIP-seq 2 (unreplicated)',
+            'Transcription factor ChIP-seq 2',
+            'Transcription factor ChIP-seq 2 (unreplicated)']
+            for pipeline in value['pipelines']):
+        yield from check_analysis_chip_seq_standards(
+            value,
+            files_structure,
+            [
+                'ChIP-seq read mapping',
+                'Raw mapping with no filtration',
+                'Pool and subsample alignments',
+                'Histone ChIP-seq',
+                'Histone ChIP-seq (unreplicated)',
+                'Transcription factor ChIP-seq',
+                'Transcription factor ChIP-seq (unreplicated)',
+                'Histone ChIP-seq 2',
+                'Histone ChIP-seq 2 (unreplicated)',
+                'Transcription factor ChIP-seq 2',
+                'Transcription factor ChIP-seq 2 (unreplicated)',
+            ],
+            '/data-standards/chip-seq/')
+
+    if any(pipeline['title'] == 'Transcription factor ChIP-seq pipeline (modERN)' for pipeline in value['pipelines']):
+        yield from check_analysis_modERN_chip_seq_standards(
+            value,
+            files_structure,
+            ['Transcription factor ChIP-seq pipeline (modERN)'],
+            '/')
+
+    '''
+    WGBS analyses
+    '''
+    if any(pipeline['title'] in [
+            'WGBS single-end pipeline',
+            'WGBS paired-end pipeline'] for pipeline in value['pipelines']):
+        yield from check_analysis_wgbs_encode3_standards(
+            value,
+            files_structure,
+            [
+                'WGBS single-end pipeline',
+                'WGBS paired-end pipeline'
+            ],
+            '/data-standards/wgbs/'
+        )
+
+    if any(pipeline['title'] == 'gemBS' for pipeline in value['pipelines']):
+        yield from check_analysis_wgbs_encode4_standards(
+            value,
+            files_structure,
+            ['gemBS'],
+            '/data-standards/wgbs/'
+        )
+
+    '''
+    ATAC-seq analyses
+    '''
+
+    if any(pipeline['title'] in [
+            'ATAC-seq (replicated)',
+            'ATAC-seq (unreplicated)'] for pipeline in value['pipelines']):
+        yield from check_analysis_atac_encode4_qc_standards(
+            value,
+            files_structure,
+            [
+                'ATAC-seq (replicated)',
+                'ATAC-seq (unreplicated)'
+            ],
+            '/atac-seq/')
+        return
+
+    '''
+    ChIA-PET analyses
+    '''
+    if any(pipeline['title'] == 'Ruan Lab ChIA-PIPE Pipeline' for pipeline in value['pipelines']):
```

Comment:
Is this pipeline title check new? The old version checks on dataset assay_term_name? I guess we are sure all Ruan lab processed data are submitted with proper pipeline?

Reply:
It is new. I think it's OK to keep this pipeline check, because all current ChIA-PET datasets with analysis objects, are also processed with this specific Ruan pipeline (https://www.encodeproject.org/report/?type=Experiment&assay_title=ChIA-PET&field=%40id&field=analysis_objects.pipelines&sort=analysis_objects.pipelines). Idan mentioned Leslie lab may also submit their own ChIA-PET data using a different pipeline, which shouldn't get audited under these standards.

Reply:
Oh. That's very good to know this is indeed specific to that pipeline. The check is then good to have.